src/AspNetCore/Example/Example.csproj

```diff
@@ -1,14 +1,24 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
     <LangVersion>latest</LangVersion>
+    <RestoreAdditionalProjectSources Condition="$(PublishNativeAot) == True">
+      https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet7/nuget/v3/index.json;
```

Comment:
dotnet7?

Reply:
Yes. Read announcement. https://github.com/dotnet/runtimelab/issues/1241#issuecomment-1026571480

Reply:
I mean app has net6 target, but you use some net7 bits. Is it OK?

Reply:
Ah, so it is only cosmetic feed changes, right?

Reply:
Yes. You can use `dotnet-experimental` but that feed would not receive fixes.

Reply:
> I mean app has net6 target, but you use some net7 bits. Is it OK?

Specifically this. It will pull just ILCompiler package. Because you don't use any `7.0.0-*` packages this library is safe. And you have to explicitly trigger additional of that feed, because I put that after condition. Safe for anybody using non-NativeAOT builds. For NativeAOT builds that's safe too. Only * package version is unsafe, but for educational purposes it is better left like this.

Reply:
ok